protobuf/api/service.proto

```diff
@@ -18,3 +21,8 @@ message CreateServiceResponse {
   string sid = 1;
   string hash = 2;
 }
+
+// GetServiceRequest defines request to retrieve a single service.
+message GetServiceRequest {
+  string hash = 1;
```

Comment:
personally, hashOrSid is a better name

Reply:
`Sid` will be replaced by a dedicated system in the future OR will be managed by CLI.